akka-samples/akka-sample-hello-kernel/build.sbt

```diff
@@ -1,12 +1,28 @@
-import NativePackagerKeys._
+import NativePackagerHelper._
 
-packageArchetype.akka_application
+name := "hello-kernel"
 
-name := """hello-kernel"""
+version := "0.1"
 
-mainClass in Compile := Some("sample.kernel.hello.HelloKernel")
+val akkaVersion = "2.3-SNAPSHOT"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-kernel" % "2.3-SNAPSHOT",
-  "com.typesafe.akka" %% "akka-actor" % "2.3-SNAPSHOT"
+  "com.typesafe.akka" %% "akka-kernel" % akkaVersion,
+  "com.typesafe.akka" %% "akka-actor" % akkaVersion,
+  "com.typesafe.akka" %% "akka-slf4j" % akkaVersion,
+  "ch.qos.logback" % "logback-classic" % "1.0.7"
 )
+
+mainClass in Compile := Some("akka.kernel.Main")
+
+enablePlugins(JavaServerAppPackaging)
+
+mappings in Universal ++= {
+  // copy additional scripts from bin directory
+  directory("bin") ++
```

Comment:
not necessary, but an example of how you can copy things if needed.
Not uncommon to have additional scripts, I think.